src/requestManager.cpp

```diff
@@ -102,15 +110,11 @@ void CRequestManager::AskFor(const CInv& obj, CNode* from, int priority)
 	  pendingTxns+=1;
 	  // all other fields are zeroed on creation
 	}
-      else  // existing
-	{
-	}
-
```

Comment:
This is my shorthand for "I have considered this case and there's nothing to do"...